QuizMaven

```diff
@@ -0,0 +1,92 @@
+QUIZ
+Note: You could verify your quiz results online https://www.tutorialspoint.com/maven/maven_online_quiz.htm but please
+be honest and answer to this questions before getting to an answer from the site provided.
+
+1.	What of the following is true about POM?
+
+•	It is fundamental Unit of Work in Maven.
+•	It is an XML file.
+•	Both of the above
+•	None of the above.
+
+ANSWER: Both of the above
+
+2.	Which of the following configuration element is present in POM.xml?
+
+•	developers
+•	mailing list
+•	Both of the above.
+•	None of the above.
+
+ANSWER: Both of the above
+
+3.	Which of the following phase in maven life cycle compiles the source code of the project?
+
+•	validate
+•	compile
+•	test
+•	package
+
+ANSWER: compile
+
+4.	Which of the following is true about Maven repository?
+
+•	A repository is a place i.e. directory where all the project jars, library jar, plugins or any other project
+specific artifacts are stored.
+•	Maven repository is used by Maven.
```

Comment:
Maven repository is used not only by Maven but by other build tools as well.
But, since word `ONLY` not used in above statement I suggest you are correct.